src/contracts/account/ProxyAccountDeployer.sol

```diff
@@ -91,7 +91,7 @@ contract ProxyAccountDeployer {
      * Creates base Account for contracts
      */
     constructor() public {
-        baseAccount = address(new ProxyAccount());
+        baseAccount = address(new ProxyAccount{ salt : keccak256("v0.1.0|BASE_ACCOUNT") }());
```

Comment:
is it worth moving the string to a public variable. then during deployment we can check that the VERSION we are using matches what is in the contract? e.g. so we dont forget to update the contracts with the new version. (ofc, could also pass it in as a parameter)

Reply:
Yeah, could be a good idea - I think this could change depending on what we do in #22